src/c78_menuINFO.hpp

```diff
@@ -136,17 +136,17 @@ void printStatusSubmenu()
 
             case 5:
                 {
-                    long lowerLimit, upperLimit;
+                    float lowerLimit, upperLimit;
                     mount.getDecLimitPositions(lowerLimit, upperLimit);
-                    lcdMenu.printMenu("DEC Lo: " + String(lowerLimit));
+                    lcdMenu.printMenu("DEC Lo: " + String(lowerLimit, 1));
                 }
                 break;
 
             case 6:
                 {
-                    long lowerLimit, upperLimit;
+                    float lowerLimit, upperLimit;
                     mount.getDecLimitPositions(lowerLimit, upperLimit);
-                    lcdMenu.printMenu("DEC Up: " + String(upperLimit));
+                    lcdMenu.printMenu("DEC Up: " + String(upperLimit, 1));
```

Comment:
Base 1? Am I missing something here?

Reply:
Ah read the wrong implementation. For floats it's the number of decimal places :woman_facepalming: